packages/blob/src/index.edge.test.ts

```diff
@@ -0,0 +1,87 @@
+import {
```

Comment:
(for my understanding) The goal of this test file is to unit test in an edge-like Jest env?

Reply:
right, I'm testing the only methods that use a different implementation on edge